lib/live_cluster/manage_controller.py

```diff
@@ -32,7 +32,47 @@
 logger.setLevel(logging.CRITICAL)
 
 
-class ManageLeafCommandController(LiveClusterCommandController):
+class LiveClusterManageCommandController(LiveClusterCommandController):
+    def pre_controller(self, line):
+        self._init_controller_arg()
+
+        if self.controller_arg is None:
+            return
+
+        mod = self.context[-1]
+
+        if mod not in self.mods:
+            self.mods[mod] = {}
+
+        # Used as the key to reference arg, normally the controllers "command".
+        try:
+            arg = line.pop(0)
+        except IndexError:
+            raise ShellException("{} is required".format(mod))
+
+        if arg not in self.modifiers | self.required_modifiers:
+            if mod not in self.mods[mod]:
+                self.mods[mod] = []
```

Comment:
Why does self.mods[mod] have a default value of {} 1st and then []?

Reply:
fixed. There were a number of things wrong here.  Not sure when that was added

Reply:
There was actually a larger problem here. I am glad you pointed it out @dwelch-spike or I would have missed it.